webapi/src/main/scala/org/knora/webapi/slice/admin/api/service/ProjectsADMRestService.scala

```diff
@@ -276,14 +275,13 @@ final case class ProjectsADMRestServiceLive(
   override def updateProjectRestrictedViewSettings(
     id: ProjectIdentifierADM,
     user: User,
-    setSizeReq: ProjectSetRestrictedViewSizeRequest
+    req: ProjectSetRestrictedViewSizeRequest
```

Comment:
The parameter. name should be also changed in docs string above.

Reply:
See db2142edc3811fdb02872355a158073213cb2927 in followup PR